src/clr-angular/data/datagrid/datagrid-hideable-column.spec.ts

```diff
@@ -56,6 +56,34 @@ export default function(): void {
         expect(context.clarityDirective.columnId).toEqual(context.clarityDirective.hideable.id);
       });
     });
+
+    describe('TypeScript Output API', function() {
+      let context: TestContext<ClrDatagridColumn<void>, HideableOutputTest>;
+
+      beforeEach(function() {
+        context = this.create(ClrDatagridColumn, HideableTest, PROVIDERS_NEEDED);
+      });
+
+      it('creates a DatagridHideableColumn instance on the DatagridColumn', function() {
+        expect(context.clarityDirective.hideable).toBeDefined();
+      });
+
+      it('defaults the HideableColumn.hidden property to false', function() {
+        expect(context.clarityDirective.hideable.hidden).toBe(false);
+      });
+
+      it('correctly populates the DatagridHideableColumn instance with an id', function() {
+        expect(context.clarityDirective.columnId).toEqual(context.clarityDirective.hideable.id);
+      });
+
+      it('emits column state change', function() {
```

Comment:
I feel bad because I asked you to add an input, and now I'm pointing out that you're missing a unit test for that input. 😂 
Well, here is the test, that way I don't feel so bad 😛:
```typescript
it('emits column state change', function() {
  context.testComponent.hidden = true;
  context.detectChanges();
  expect(context.clarityDirective.hideable.hidden).toBeTrue();
});
```

Reply:
No problem at all, thanks for providing it 😄 It's included now